### PR TITLE
Implement select all layers shortcut

### LINKED ISSUE #219: Ctrl+A to select all layers

### PR DESCRIPTION
**Pull request by AyeTbk**
Closes #219

---
This change is [Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/242)

### CODE REVIEW

File: client/web/src/components/widgets/inputs/MenuBarInput.vue [hunk 1]
```diff
@@ -106,6 +106,7 @@ const menuEntries: MenuListEntries = [
 			[
 				{ label: "Undo", shortcut: ["Ctrl", "Z"], action: async () => (await wasm).undo() },
 				{ label: "Redo", shortcut: ["Ctrl", "⇧", "Z"] },
+				{ label: "Select all layers", shortcut: ["Ctrl", "A"], action: async () => (await wasm).select_all_layers() },
```
Comment (Keavon) [inline client/web/src/components/widgets/inputs/MenuBarInput.vue:109]: Can you add a new section between "Document" and "View" called "Layer", and put this in it? Also please use Title Case for "Select All Layers".

**Review by AyeTbk — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/242)* status: 0 of 4 files reviewed, 2 unresolved discussions (waiting on @Keavon)

___
*[client/web/src/components/widgets/inputs/MenuBarInput.vue, line 109 at r1](https://reviewable.io/reviews/graphiteeditor/graphite/242#-MeHFs0W9Bcfep1PC9WJ-r1-109:-MeHL_zS2zsxL8eMJ_IK:b-896fix) ([raw file](https://github.com/graphiteeditor/graphite/blob/0e102c82dce2f301b0f1ef2da8c9f1465ad0a838/client/web/src/components/widgets/inputs/MenuBarInput.vue#L109)):*
<details><summary><i>Previously, Keavon (Keavon Chambers) wrote…</i></summary><blockquote>

Can you add a new section between "Document" and "View" called "Layer", and put this in it? And then just name the entry "Select All" (notice that it's Title Case).
</blockquote></details>

Done.
___
*[core/editor/src/document/document_message_handler.rs, line 300 at r2](https://reviewable.io/reviews/graphiteeditor/graphite/242#-MeHaHe45gCkmEj5LsUb:-MeHaHe45gCkmEj5LsUc:b-ic92lp) ([raw file](https://github.com/graphiteeditor/graphite/blob/3650a1c7f1595629a99dce95ff9aec3a34c069e6/core/editor/src/document/document_message_handler.rs#L300)):*
> ```XML
> 				self.clear_selection();
> 				let children = self.active_document_mut().layer_panel(&[]).expect("The provided Path was not valid");
> 				responses.push_back(FrontendMessage::ExpandFolder { path: vec![], children }.into());
> ```

I'm not quite sure this is correct, but it does seem to work. I need to send a message so the UI knows something changed. as I understand it, but I couldn't find one that seemed perfect for the task, so I just replicated what is done for select_layer().

**Review by Keavon — APPROVED** [review]
> Reviewed 4 of 4 files at r2.
> *[Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/242)* status: all files reviewed, 1 unresolved discussion (waiting on @AyeTbk)

___
*[core/editor/src/document/document_message_handler.rs, line 300 at r2](https://reviewable.io/reviews/graphiteeditor/graphite/242#-MeHaHe45gCkmEj5LsUb:-MeHdKwfAp6fTp-9bt1o:b-1koudw) ([raw file](https://github.com/graphiteeditor/graphite/blob/5ca9de2e74df1dcfec88d00359c6a98292783c3a/core/editor/src/document/document_message_handler.rs#L300)):*
<details><summary><i>Previously, AyeTbk (Simon Desloges) wrote…</i></summary><blockquote>

I'm not quite sure this is correct, but it does seem to work. I need to send a message so the UI knows something changed. as I understand it, but I couldn't find one that seemed perfect for the task, so I just replicated what is done for select_layer().
</blockquote></details>

That's correct! ExpandFolder re-sends the current contents of the folder, and that is the way we designed it for folders to have content updated from the backend to the frontend.

**Review by Keavon — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/242)* status: all files reviewed, 1 unresolved discussion (waiting on @AyeTbk)

**Review by Keavon — APPROVED** [review]
> Reviewed 4 of 4 files at r3.
> *[Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/242)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @AyeTbk)

**Pull request merged**